godtools/AppDelegate.swift

```diff
@@ -16,7 +16,7 @@ import RealmSwift
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-    var flowController: BaseFlowController?
+    var flowController: PlatformFlowController?
```

Comment:
I understand this because of the function `goToUniversalLinkedResource`, but did you test this @ryancarlson and did you find no problems when using the app? Was concerned about this, but only a test to see if everything is showing up will be enough.

Reply:
Ah, that should probably still be `BaseFlowController`. In another PR I'm introducing `ArticleFlowController` for the Questions about God articles. I'll look at this some more.

Reply:
Oh ok, yes so in that way maybe will require a small refactoring.

Reply:
Fixed, thanks!